src/officehours_api/notifications.py

```diff
@@ -12,10 +12,18 @@
 
 from officehours_api.models import Queue, Meeting, MeetingStatus
 
-logger = logging.getLogger(__name__)
-
-twilio = TwilioClient(settings.TWILIO_ACCOUNT_SID, settings.TWILIO_AUTH_TOKEN)
-
+def initialize_twilio():
+    logger = logging.getLogger(__name__)
+
+    if (settings.TWILIO_ACCOUNT_SID and settings.TWILIO_AUTH_TOKEN and settings.TWILIO_MESSAGING_SERVICE_SID):
+        twilio_client = TwilioClient(settings.TWILIO_ACCOUNT_SID, settings.TWILIO_AUTH_TOKEN)
+        logger.info("Twilio client initialized.")
+    else:
+        logger.info("Twilio client setup skipped. Twilio settings values are not set (TWILIO_ACCOUNT_SID, TWILIO_AUTH_TOKEN, TWILIO_MESSAGING_SERVICE_SID).")
+        twilio_client = None
+    return logger, twilio_client
+
+logger, twilio = initialize_twilio()
```

Comment:
Is there any further error handling needed when the `twilio` value is None?

Reply:
I’ve added a new error type to address this case when calling on “send otp” while twilio settings are not set. Wasn’t super sure if this was the best way to raise an exception but it works for now.